Bio/PDB/MMCIFIO.py

```diff
@@ -0,0 +1,301 @@
+"""Write an mmCIF file.
+See https://www.iucr.org/resources/cif/spec/version1.1/cifsyntax for syntax.
+"""
+
+import re
+from collections import defaultdict
+
+from Bio._py3k import basestring
+from Bio.PDB.StructureBuilder import StructureBuilder
+from Bio.PDB.PDBIO import Select
+
+# If certain entries should have a certain order of keys, that is specified here
+mmcif_order = {
+    "_atom_site": [
+        "group_PDB",
+        "id",
+        "type_symbol",
+        "label_atom_id",
+        "label_alt_id",
+        "label_comp_id",
+        "label_asym_id",
+        "label_entity_id",
+        "label_seq_id",
+        "pdbx_PDB_ins_code",
+        "Cartn_x",
+        "Cartn_y",
+        "Cartn_z",
+        "occupancy",
+        "B_iso_or_equiv",
+        "pdbx_formal_charge",
+        "auth_seq_id",
+        "auth_comp_id",
+        "auth_asym_id",
+        "auth_atom_id",
+        "pdbx_PDB_model_num",
+    ]
+}
+
+
+class MMCIFIO(object):
```

Comment:
You have a module called ``MMCIFIO`` and a class called ``MMCIFIO`` which is a very bad idea as they clash and will be a source of confusion depending on how they are imported.

This is one reason why PEP8 says use lower case for module names and title case for classes https://www.python.org/dev/peps/pep-0008/#naming-conventions

I appreciate that ``Bio.PDB`` breaks those rules (as do other bits of Biopython), but it is historic code and we're stuck with it for backwards compatibility.

I would nevertheless suggest considering using ``Bio/PDB/mmcifio.py`` for the filename (but am interested to hear what other people would recommend here).

Reply:
Yes I see the issue. Changing the filename to `mmcifio.py` seems sensible and would be in line with PEP8 conventions, and this is probably worth inconsistency within the PDB module.